index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <title>TSE training project</title>
+</head>
+
+<body>
+    <header>
+        <h3 class="header-color">
```

Comment:
Instead of the <h3> tag, you can have the <h1> tag since it is the main title of the page as this is critical to good technical SEO.

Reply:
HI, @lily-coder thank you for your valuable comment. I will work on it.